app/main.py

```diff
@@ -7,20 +9,30 @@
     MEDIA_PATH, STATIC_PATH, templates)
 from app.routers import agenda, event, profile, email
 
+from app.internal.logger_customizer import LoggerCustomizer
+
 
 models.Base.metadata.create_all(bind=engine)
 
 app = FastAPI()
 app.mount("/static", StaticFiles(directory=STATIC_PATH), name="static")
 app.mount("/media", StaticFiles(directory=MEDIA_PATH), name="media")
 
+# Configure logger
+config_path = Path(__file__).parent / 'internal'
```

Comment:
Use our already exists configuration file. Please don't create a separate file for logging configuration.